packages/k2-alpine/src/components/GroupList/GroupList.stories.tsx

```diff
@@ -49,6 +50,25 @@ export const All = (): JSX.Element => {
               onPress: () => {
                 showAlert({ title: 'Pressed', buttons: [{ text: 'OK' }] })
               }
+            },
+            {
+              title: 'Title 5',
+              value: isAccordionExpanded ? 'Expanded' : 'Collapsed',
+              onPress: () => {
+                setIsAccordionExpanded(!isAccordionExpanded)
+              },
+              accordion: {
+                expanded: isAccordionExpanded,
```

Comment:
same here, expanded should be internal. if we want to we can pass an initial value for expanded.